api/mesh/v1alpha1/traffic_route.proto

```diff
@@ -222,6 +219,5 @@ message TrafficRoute {
   }
 
   // Configuration for the route.
-  Conf conf = 3
-      [ (validate.rules).message.required = true ];
+  Conf conf = 3 [ (validate.rules).message.required = true ];
```

Comment:
This is weird right? Why is this changing exactly? I don't think it needs to be in this PR, I don't seem to get this running `make check` locally.

Reply:
yeah, make format is different on CI and my local machine :|

Reply:
I'm reverting the commit to see if it passes CI.

Reply:
https://github.com/kumahq/kuma/pull/10742